lib/disability_compensation/providers/document_upload/lighthouse_supplemental_document_upload_provider.rb

```diff
@@ -102,28 +109,24 @@ def log_upload_attempt
     StatsD.increment("#{@statsd_metric_prefix}.#{STATSD_PROVIDER_METRIC}.#{STATSD_ATTEMPT_METRIC}")
   end
 
-  def log_upload_success(lighthouse_request_id)
+  def log_upload_success(lighthouse_document_request_id)
     Rails.logger.info(
       'LighthouseSupplementalDocumentUploadProvider upload successful',
       {
         **base_logging_info,
-        lighthouse_request_id:
+        lighthouse_document_request_id:
```

Comment:
Changed to make these clearer; this is the ID Lighthouse returns to us on a successful upload so we can query their polling endpoint to check on the processing status